playbooks/support/README.md

```diff
@@ -19,111 +50,167 @@ which is determined at least a month in advance (giving people ample time to mak
+
+- How many people (or partners, etc.) does this affect?
+- Have you been able to reproduce?
+- Is this a new problem or something that we are aware of?
+
+3. Update the severity on the Jira Ops ticket. See [severity of incidents](#severity-of-incidents) below.
+4. If the reported incident does not qualify for an immediate response:
+
+- Direct stakeholders in the slack thread to a product team’s channel to ensure that a Jira Issue is tracked for
+  this possible issue
+- Link the Jira Ops ticket to any relevant bugs that are tracked (and create an additional one if necessary)
+- Mark the ticket as **CANCELED**
 
-#### 1. Be available to answer requests and respond to immediate issues
+### Step 4: Address Incident
 
-- On-call is a 24/7 responsibility (for critical issues). The non-work hours can be split across the on-call
-  engineers and over timezones where appropriate.
-- Monitor the [#incidents slack channel](https://artsy.slack.com/messages/C9RK0BLEP/) 🔒 for critical issues.
-- Look out for trends and create bugs (in the
-  [universal bug backlog](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=34)) where necessary
-  that can be triaged into individual teams' sprints by a PM.
-- If the issue is obviously a non-incident, take the time to educate the reporter on the desired process (see the [examples](#examples) below). You may point them to the [Guide to Reporting Bugs](https://www.notion.so/artsy/Guide-to-reporting-bugs-cc25e1ff41194228b476c4963c646817) doc in Notion which describes this in a general way.
+#### Investigate --> Identify --> Escalate --> Fix
 
-#### 2. Investigate and address critical issues
+Discuss the incident's resolution in the dedicated slack channel.
 
-- Acknowledge the issue in slack (escalate to the [#incidents](https://artsy.slack.com/messages/C9RK0BLEP/) 🔒 channel
-  if not already there)
-- **Use slack threads** to keep the channel and its notifications focused, and allow others to monitor for new
-  incidents at a glance.
-- Investigate the issue individually and as a pair.
-- Judge the severity (based on intuition or a helpful PM/business owner)
 - Fix if possible using [accumulated playbooks (wiki)](https://github.com/artsy/potential/wiki) 🔒.
 - If unable to fix using shared resources, contact the relevant [point-person](#point-people) for help and pair
-  with them on a fix. Be sure to contribute lessons back to shared documentation. **You are not expected to know the ins and outs of every system, so don't hesitate to involve the wider team.**
-- Point-people should only be contacted during off-hours for outages that the on-call engineers are unable to
-  diagnose.
+  with them on a fix. Be sure to contribute lessons back to shared documentation. **You are not expected to know
+  the ins and outs of every system, so don't hesitate to involve the wider team.**
+  - Point-people should only be contacted during off-hours for outages that the on-call engineers are unable to
```

Comment:
This might not be necessary, if we've correctly defined "incidents" elsewhere and when to escalate elsewhere.